colossalai/fx/profiler/memory.py

```diff
@@ -73,9 +73,9 @@ def calculate_fwd_tmp(n: Node) -> int:
 
     def is_relu_node(n: Node) -> bool:
         if n.op == 'call_function':
-            return n.target in [torch.nn.functional.relu]
+            return n.target in [torch.nn.functional.relu, torch.nn.functional.softmax]
         elif n.op == 'call_module':
-            return type(n.graph.owning_module.get_submodule(n.target)) in [torch.nn.ReLU]
+            return type(n.graph.owning_module.get_submodule(n.target)) in [torch.nn.ReLU, torch.nn.Softmax]
```

Comment:
as I mentioned, softmax is another relu_like node

Reply:
I think this is gonna confuse the future maintainer, what is a relu_like node?

Reply:
I will think about this later. Unfortunately, I am also confused lol. Maybe I should add this to `constants.py`?

Reply:
at least you should define this category of ops so that its semantics is clear.

Reply:
Okay, I will make some comments there